src/core/basetypes/MCDataDecoderUtils.cpp

```diff
@@ -0,0 +1,189 @@
+#include "MCDataDecoderUtils.h"
+
+#include <libetpan/libetpan.h>
```

Comment:
could also #include <strings.h> and #include <string.h> to unbreak the build on linux?